docs/topics/multiplatform/multiplatform-ios-integration-overview.md

```diff
@@ -0,0 +1,67 @@
+[//]: # (title: iOS integration methods)
+
+You can integrate a Kotlin Multiplatform shared module into your iOS app. For that, an [iOS framework](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPFrameworks/Concepts/WhatAreFrameworks.html)
+is generated from the shared module. Then, you should add it as a dependency to the iOS project:
```

Comment:
It's not entirely clear at which point or how the framework is generated. Maybe "you can generate..."? No additional details required, but there's no connotation of waiting for something to happen.